prow/cluster/jobs/istio/installer/istio.installer.master.yaml

```diff
@@ -0,0 +1,52 @@
+job_template: &job_template
+  branches:
+  - "^master$"
+  decorate: true
+  path_alias: istio.io/istio
+
+istio_container: &istio_container
+  image: gcr.io/istio-testing/istio-builder:v20190624-478286dc
+  # Docker in Docker
+  securityContext:
+    privileged: true
+  resources:
+    requests:
+      memory: "2Gi"
+      cpu: "3000m"
+    limits:
+      memory: "24Gi"
+      cpu: "3000m"
+
+istio_container_with_kind: &istio_container_with_kind
```

Comment:
You can just delete this istio_container_with_kind section since the istio_container already has KinD package by default.

Reply:
thanks this was C&P from istio - but makes complete sene now :)